packages/loader/container-loader/src/container.ts

```diff
@@ -1722,7 +1722,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         const local = this.clientId === message.clientId;
 
         // Forward non system messages to the loaded runtime for processing
-        if (!isSystemMessage(message)) {
+        if (isRuntimeMessage(message)) {
```

Comment:
@vladsud Do we need to worry about back-compat for legacy op format?
(see `containerRuntime.ts`:371

Reply:
I'm not sure what exactly you mean here. Your pointer to RuntimeMessage (I assume) is not relevant here, as these two namespaces do not overlap. RuntimeMessage ops are what used within runtime, and such ops are always wrapped into type: "op" op at this (loader) level

Reply:
I might be wrong, but if I read it correctly, before PR #2496, you would have sent the set of `RuntimeMessage` in the `message.type` directly instead of wrapped.  Now, this would filter that out, causing back-compat issue for old file content?

Reply:
@vladsud what's your thought here?